lib/extensions/ar_base.rb

```diff
@@ -29,7 +29,7 @@ def self.vacuum
 
     def self.connectable?
       connection && connected?
-    rescue ActiveRecord::NoDatabaseError, PG::ConnectionBad
+    rescue ActiveRecord::ConnectionNotEstablished, ActiveRecord::NoDatabaseError, PG::ConnectionBad
```

Comment:
I think this is rails 6.0 safe.  Testing it right now.  This constant exists back then.

Reply:
```
Loading development environment (Rails 6.0.5.1)
irb(main):001:0> ActiveRecord::ConnectionNotEstablished
=> ActiveRecord::ConnectionNotEstablished
```

Reply:
Oh good call